### DISCUSSION TITLE
how to use on  windwos mysys [package] [build]

### DISCUSSION THREAD

**Discussion by stan1334** [zeroc-ice/ice#2229] [Q&A] [answered]
### Version

windows10
x64
mysys+mingw64+gcc

pacman -S zeroc-ice

/etc/pacman.conf add:
[zeroc]
Include = /etc/pacman.d/mirrorlist.ice

mirrorlist.ice:
Server = https://download.zeroc.com/Ice/3.6/el7/zeroc-ice3.6.repo

pacman -Sy zeroc-ice*
:: Synchronizing package databases...
 clangarm64 is up to date
 mingw32 is up to date
 mingw64 is up to date
 ucrt64 is up to date
 clang32 is up to date
 clang64 is up to date
 msys is up to date
 zeroc.db failed to download
error: failed retrieving file 'zeroc.db' from zeroc.com : The requested URL returned error: 403
error: failed to synchronize all databases (unexpected error)

i think server url is err。
Could you please provide the ice source address of mysys?

if havent repo，i only build sourceCode on mysys windows10 。 

@pepone
@bentoi

### Describe the bug

how to use ice on mysys windwos

### Expected behavior

add repo 
install  ice-all-runtime ice-all-devel

**Comment by pepone** [comment, marked as answer]
Hi,

We don't provide any MinGW packages, with 3.6 we supported building Ice with MinGW as a means of supporting Ice for Ruby extension on Windows see https://github.com/zeroc-ice/ice/blob/3.6/cpp/BuildInstructionsMinGW.md

This was later removed in the 3.7 release. If you are developing an Ice application on Windows you should use Ice 3.7 and one of the supported compilers. See https://doc.zeroc.com/ice/3.7/release-notes/supported-platforms-for-ice-3-7-10#id-.SupportedPlatformsforIce3.7.10v3.7-IceforC++

Cheers,
Jose
